azure_arc_k8s_jumpstart/aksengine/arc_k8s_aksengine.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Variables
+location=westeurope
+k8s_rg=aksengine
+aksengine_cluster_file="/tmp/aksengine_cluster.json" 
+aksengine_vm_size=Standard_B2ms
+domain=abc$RANDOM
+output_dir=/tmp/aksengine
+keyvault_name=yourKeyvault
+arc_rg=k8sarc
+arc_name=myaksengine
```

Comment:
Change this param to something more descriptive and update references in the README